tests/e2e-tests/jest.e2e.config.ts

```diff
@@ -24,7 +24,7 @@ const config: Config.InitialOptions = {
     ]
   ],
   setupFiles: ['<rootDir>/../shared-test-code/setup/setup.ts'],
-  testPathIgnorePatterns: ['/src/', '/tests/integration-tests/'],
+  testMatch: ['<rootDir>/*.spec.ts'],
```

Comment:
Have you checked that just the e2e tests get run now when using this test config? It looks to me like this might just run everything with spec.ts in its name, which is obviously not what you want!

Reply:
FIxed! I did some git jiggery pokery and was missing a change. Updated to `<rootDir>/**/*.spec.ts`

`<roorDir>` is the directory where the config file sits so all good on that front!